docs/topics/device_connection.md

```diff
@@ -0,0 +1,265 @@
+
+    POST /iot/services
+    Content-Type: application/json
+    Fiware-service: smartown
+    Fiware-servicepath: /gardens
+
+    {
+      "services": [
+        {
+          "protocol": [
+                  "IoTA-UL"
+                ],
+          "apikey": "801230BJKL23Y9090DSFL123HJK09H324HV8732",
+          "entity_type": "Device"
+        }
+      ]
+    }
+
+This request specifies that the default type for the Context Entities representing our devices will be "Device", and
+that the protocol we will use for our devices will be Ultralight 2.0.
+
+### Provisioning the device
+
+The next step is provisioning the device in the platform. In this example, we will provision a single device: a Weather
+Station having two sensors, a humidity sensor and a thermometer. We can also use this provisioning to add some static
+information we know for our entity, that need not be reported by the device itself, but we want appearing in the
+entity itself. As en example, we will provision statically the Reference of the Weather Station and its location.
+
+To provision the device in the platfor we issue the following request to the IoTAgent:
+
+    POST /iot/devices
+    Content-Type: application/json
+    Fiware-service: smartown
+    Fiware-servicepath: /gardens
+
+    {
+      "devices": [
+        {
+          "device_id": "ws1956672",
+          "protocol": "IoTA-UL",
+          "entity_name": "WeatherStation1",
+          "entity_type": "Device",
+          "attributes": [
+            {
+              "object_id": "t",
+              "name": "temperature",
+              "type": "degrees"
+            },
+            {
+              "object_id": "h",
+              "name": "humidity",
+              "type": "percentage"
+            }
+          ],
+          "commands": [
+            {
+              "name": "configuration",
+              "type": "command"
+            }
+          ],
+          "static_attributes": [
+            {
+              "name": "location",
+              "type": "geo:point",
+              "value": "22.1, -17"
+            },
+            {
+              "name": "reference",
+              "type": "string",
+              "value": "917834508965243"
+            }
+          ]
+        }
+      ]
+    }
+
+This will provision our ws1 workstation in the system, making it ready to receive measures.
+
+### Sending the measure
+
+In order to send measures to the platform, the device has to fulfill the following requirements:
+
+* Connectivity to the South Bound of the platform.
+* Either an MQTT or an HTTP client (as those are the transport protocols currently supported).
+
+Being the most common, we will assume that your constraint device have an HTTP client, that can send requests to the
+platform. This interaction with the Platform can be implemented with whichever programming language your device supports.
+No SDKs are provided for south bound interactions.
```

Comment:
I'd suggest to add a "At the present moment, no sdk..." in order to make this softer.